controllers/projects_controller.go

```diff
@@ -107,38 +102,15 @@ func (u *Projects) Create(c web.C, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	logging.SharedInstance().MethodInfo("ProjectsController", "Create").Debugf("post new project parameter: %+v", newProjectForm)
-	project := projectModel.NewProject(0, current_user.Id, newProjectForm.Title, newProjectForm.Description, sql.NullInt64{})
-	if !project.Save() {
+
+	project, err := projectModel.Create(current_user.Id, newProjectForm.Title, newProjectForm.Description, newProjectForm.RepositoryID, newProjectForm.RepositoryOwner, newProjectForm.RepositoryName, current_user.OauthToken)
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: projectModel
* [gotype] invalid operation: current_user (variable of type *invalid type) has no field or method Id
* [gotype] invalid operation: current_user (variable of type *invalid type) has no field or method OauthToken